lib/pay_day_loan.ex

```diff
@@ -182,11 +279,21 @@ defmodule PayDayLoan do
 
   @doc """
   Check for cached pid, but do not request a load
+
+  This is a legacy API method and may be deprecated.  Use `peek/2`.
   """
   @spec peek_pid(pdl :: t, key :: PayDayLoan.key)
   :: {:ok, pid} | {:error, :not_found}
   def peek_pid(pdl = %PayDayLoan{}, key) do
-    CacheStateManager.get_pid(pdl.cache_state_manager, key)
+    peek(pdl, key)
```

Comment:
That sounds fine to me. I was thinking of a runtime deprecation warning in any case, just to help in case we are calling it somewhere that we miss.